R/checkMissings.R

```diff
@@ -1,20 +1,20 @@
 #### Check consistency of missings
 #############################################################################
-#' Check and Adjust Missing Coding
+#' Check and Adjust Missing Tags
 #'
-#' Functions to check if missings are coded and labeled correctly in a \code{GADSdat} object.
+#' Functions to check if missings are tagged and labeled correctly in a \code{GADSdat} object.
 #'
 #' \code{checkMissings()} compares value labels (\code{valLabels}) and missing tags (\code{missings}) of a \code{GADSdat} object and its
 #' meta data information.
-#' \code{checkMissingsByValues()} compares labeled values (\code{value}) and missing codes (\code{missings}) of a \code{GADSdat} object
+#' \code{checkMissingsByValues()} compares labeled values (\code{value}) and missing tags (\code{missings}) of a \code{GADSdat} object
 #' and its meta data information.
 #' Mismatches are reported and can be automatically adjusted. Note that all checks are only applied to the
 #' meta data information, not the actual data. For detecting missing value labels, see \code{\link{checkMissingValLabels}}.
 #'
 #'@param GADSdat \code{GADSdat} object imported via \code{eatGADS}.
 #'@param missingLabel Single regular expression indicating how missing labels are commonly named in the value labels.
 #'@param missingValues Numeric vector of values which are commonly used for missing values.
-#'@param addMissingCode If \code{TRUE}, missing codes are added according to  \code{missingLabel} or \code{missingValues}.
+#'@param addMissingCode If \code{TRUE}, missing tags are added according to  \code{missingLabel} or \code{missingValues}.

```

Comment:
Nit: There is a superfluous space here:

`to _\code{missingLabel}`

Reply:
Eagle eye! 🦅